python/cdp_san_jose_backend/clean_database.py

```diff
@@ -0,0 +1,48 @@
+from cdp_backend.database import models as db_models
+import fireo
+from google.auth.credentials import AnonymousCredentials
+from google.cloud.firestore import Client
+
+# Connect to the database
+fireo.connection(client=Client(
+    project="cdp-san-jose-5d9db455",
+    credentials=AnonymousCredentials()
+))
+
+# Read from the database
+sessions = db_models.Session.collection.fetch(1000)
+
+i=0
+for session in sessions:
+    if session.session_datetime.year == 2020 and session.session_datetime.month==1 and session.session_datetime.day==7:
+        print("Session_time=" + str(session.session_datetime) + " ID=" + str(session.id) + " key=" +str(session.key) +" _update_time=" + str(session._update_time))
+    i=i+1
+
+print("----")
+
+dictionary = dict()
+sessions_for_delete = []
+sessions = db_models.Session.collection.fetch(1000)
+for session in sessions:
+    if session.session_datetime not in dictionary:
+        dictionary[session.session_datetime] = []
```

Comment:
Nothing wrong here, but these 2 lines could be removed if `dictionary` is a `collections.defaultdict(list)`.

Reply:
Eh -- I prefer this how it is -- but then again, I am just bad at using defaultdict stuff.

Reply:
Thanks!